ipc/socket/socket_factory_spawn_test.go

```diff
@@ -264,10 +288,14 @@ func Test_Unix_Timeout2(t *testing.T) {
 
 	cmd := exec.Command("php", "../../tests/slow-client.php", "echo", "unix", "200", "0")
 
-	w, err := NewSocketServer(ls, time.Millisecond*100, log).SpawnWorkerWithContext(context.Background(), cmd)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Millisecond*100)
+	defer cancel()
+
+	w, err := NewSocketServer(ls, time.Millisecond*100, log).SpawnWorkerWithContext(ctx, cmd)
```

Comment:
Relay timeout is not the same as allocate (spawn) timeout (in the sense of passing so small ctx. Because GitHub has kind of slow runners, this change may lead to the flaky test). Furthermore, `NewSocketServer` timeout can be removed, since the associated method `findRelay` which consumed this timeout was removed accordingly.

Reply:
`findRelay` was removew, but `findRelayWithContext` has present. Why i should remove ctx?

Reply:
> NewSocketServer timeout can be removed

Not context, timeout. Context time should be increased.